src/main/io/displayport_max7456.c

```diff
@@ -166,12 +166,17 @@ static const displayPortVTable_t max7456VTable = {
 
 displayPort_t *max7456DisplayPortInit(const vcdProfile_t *vcdProfile)
 {
-    displayInit(&max7456DisplayPort, &max7456VTable);
 #ifdef USE_OSD_SLAVE
-    max7456Init(max7456Config(), vcdProfile, false);
+    if (!max7456Init(max7456Config(), vcdProfile, false))
 #else
-    max7456Init(max7456Config(), vcdProfile, systemConfig()->cpu_overclock);
+    if (!max7456Init(max7456Config(), vcdProfile, systemConfig()->cpu_overclock))
```

Comment:
Can we have these as

Reply:
You mean balanced number of parens?

Reply:
Done

Reply:
Well, the parens were already balanced the way you did it - just not the ifs. :wink: